catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/ListMetacardTypeImpl.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.workspace;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.impl.MetacardTypeImpl;
+import java.util.HashSet;
+import java.util.Set;
+
+public class ListMetacardTypeImpl extends MetacardTypeImpl {
+
+  public static final String LIST_TAG = "list";
+
+  public static final String LIST_METACARD_TYPE_NAME = "metacard.list";
+
+  public static final String LIST_CQL = "list.cql";
+
+  public static final String LIST_ICON = "list.icon";
+
+  public static final String LIST_BOOKMARKS = "list.bookmarks";
+
+  private static final Set<AttributeDescriptor> LIST_DESCRIPTORS;
+
+  static {
+    LIST_DESCRIPTORS = new HashSet<>();
+
+    LIST_DESCRIPTORS.add(
+        new AttributeDescriptorImpl(
+            Metacard.ID,
```

Comment:
We could mix these in with the Core types instead of having to redefine them, though this should still work I believe (via the constructor on metacard type https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardTypeImpl.java#L112-L116)